src/components/ViewParty/ViewParty.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useContext } from 'react';
+import { Table, Button, Accordion, Card } from 'react-bootstrap';
+import cookies from 'js-cookie';
+import './ViewParty.css'
+import { UserContext } from '../../UserContext'
+import axios from 'axios';
+
+//temporary metadata until we can pull it from the DB
+const tempPartyList = [
```

Comment:
You can actually remove this list. ViewParty component will work better as a component that accepts props.

Reply:
Handled, got rid of the list.